lib/document-to-html.js

```diff
@@ -379,13 +389,21 @@ function DocumentConversion(options, comments) {
         ];
     }
 
-    function createDeferedChart(element, messages, options){
+    function createDeferredChart(element, messages, options){
         return element.read().then(
             function (value) {
                 return Html.freshElement('pwl:chart', null, [Html.text('Chart')]);//[Html.text(JSON.stringify(value))]);
             }
         );
     }
+	function createDeferredFooter(element, messages, options) {
```

Comment:
This being passed footer.value - is that an element and one that can be read() ?